databricks/koalas/typedef/typehints.py

```diff
@@ -128,8 +131,18 @@ def as_spark_type(tpe) -> types.DataType:
         return types.FloatType()
     elif tpe in (np.int32, "int32", "i"):
         return types.IntegerType()
-    elif tpe in (int, np.int, np.int64, "int", "int64", "long", "bigint"):
+    elif tpe in (int, np.int, np.int64, "int", "int64", "long"):
         return types.LongType()
+    elif isinstance(tpe, str) and tpe in ("bigint",):
```

Comment:
I think we can even remove this and add this in the release note as a breaking change since `bigint` isn't really a numpy type, and we did not document that `astype` can take DDL formatted type Spark types. But deprecating is fine too. I will leave it to you.

Reply:
Then shall we remove this for now.